### ISSUE TITLE
It is unstable using your new release code in 64*64 resolution

### ISSUE THREAD

**Issue by akafen** [github-pengge/PyTorch-progressive_growing_of_gans#10]
It is unstable using your newest release code in 64*64 resolution,why?

**Comment by github-pengge** [comment]
Can't figure out why. In my experiment, when fading in 256x256, the model crashed.

**Comment by github-pengge** [comment]
Try `train_no_tanh.py` with latest `data.py` script.

**Comment by yuanzhaoYZ** [comment]
yeah, same thing happened to me as well. In my case, the model collapsed very early on at `16*16 fade in`. ( I used default value for all the hyper-parameters listed in the README)

**Comment by ZhilingYe** [comment]
Me too, the model coIlapsed very early. BTW, the model is trained on my own data set. Is there something important??

**Comment by github-pengge** [comment]
Use `train_no_tanh.py` instead.

**Comment by ZhilingYe** [comment]
Thx!! It works. I can successfully obtain meaningful images.
